torch/autograd/grad_mode.py

```diff
@@ -253,3 +254,39 @@ def __exit__(self, *args) -> None:
 
     def clone(self):
         return self.__class__(self.mode)
+
+
+class _set_view_replay_enabled(_DecoratorContextManager):
```

Comment:
Could you rename this to `_force_original_view_tracking()` or something like that to be closer to what it does?